### DISCUSSION TITLE
How to deal with water bodies?

### DISCUSSION THREAD

**Discussion by calvarezgarreton** [mhm-ufz/mHM#54] [Q&A]
Hi, 
I am modelling some basins in Chile were lakes cover almost 15% of their total areas. 

If I leave the water bodies pixels as pervious land cover classes, they won't be able to evaporate all the PET given the soil moisture restrictions and vegetation parameterisation of those pixels. This could potentially lead to large ET errors if the water body is large and mHM is calibrated using a downstream gauge (the total Evaporation from the lake should be ~ Area_lake x PET, always larger than Area_lake x ET_mHM). I am currently correcting PET with LAI ( !> potential evapotranspiration (PET)!>  -1) and thought on prescribing very large LAI values for those pixels as a workaround, but this does not solve the soil moisture restrictions.

Do you have any suggestions on how to deal with this? Or any mHM application you know of that I could look at?

Thanks!
Camila

**Comment by StephanThober** [comment]
Hi Camila,

yes, this is indeed an issue where currently only workarounds do exist. In an ongoing project, I removed the lakes completely from the dem of mHM and provided P - PET as input to the routing component (currently also not implemented in develop). However, that also does not solve the issue that water is actually evaporating from lake areas. Just for clarification, which variable are you interested more in: ET or Q?

Best,
Stephan
